Formula/handbrake.rb

```diff
@@ -0,0 +1,28 @@
+class Handbrake < Formula
+  desc "Open-source video transcoder available for Linux, Mac, and Windows."
+  homepage "https://handbrake.fr/"
+  url "https://github.com/HandBrake/HandBrake.git", :revision => "72e77efa1c309be6a5e525a9484ec131707af848"
+  version "1.0.0-alpha1"
+  head "https://github.com/HandBrake/HandBrake.git"
+
+  depends_on "autoconf" => :build
+  depends_on "automake" => :build
+  depends_on "cmake" => :build
+  depends_on "libtool" => :build
+  depends_on "pkg-config" => :build
+  depends_on "yasm" => :build
```

Comment:
yes. I chatted with @ilovezfs about it. They have crazy upstream projects (where they do report bugs to that regularly break stuff in minor point updates that are not detectable in tests (at east automatically). I made sure and provided a pull to them which got pulled in by now to at least verify the downloads of the libs via sha256